src/config/test/translationService.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { TranslationService, TranslationsData } from "../translationService";
```

Comment:
👍  for the tests
We should test `getImageSource` too

Reply:
Fixed @ [76b1d3c](https://github.com/genexuslabs/web-standard-functions/pull/65/commits/76b1d3c04b52ad84b43ada998a2365ab1408db1e)